Fotoblog.DAL/Entities/TagEntity.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace Fotoblog.DAL.Entities
+{
+    public class TagEntity
+    {
+        public int Id { get; set; }
+        public string Name { get; set; } = string.Empty;
+        public ICollection<PhotoEntity> ?Photos { get; set; }
```

Comment:
Użyj List zamiast ICollection. Nullable też niepotrzebne. Jeśli dobrze pamiętam kolekcje są nullable z defaultu (mogę się mylić).

Reply:
bez ? lista to też non nullable property, ale można przypisać od razu pustą listę